djangoproject/templates/conduct/reporting.html

```diff
@@ -111,14 +111,17 @@ <h3>{% translate "What if your report concerns a possible violation by a committ
       may not feel comfortable sending your report to the committee, as all members will
       see the report.{% endblocktranslate %}</p>
 
-  <p>
-    {% blocktranslate trimmed %}
-      In that case, you can make a report directly to any or all of the current
-      (vice/co) chairs of the Code of Conduct committee. Their e-mail addresses are
-      listed on the <a href="/foundation/committees/">Code of Conduct Committee</a>
-      page. The chairs will follow the usual enforcement process with the other
-      members, but will exclude the member(s) that the report concerns from any
-      discussion or decision making.{% endblocktranslate %}</p>
+  {% url 'members:teams' as teams_url %}
+  {% with coc_teams_url=teams_url|add:'#code-of-conduct-team' %}
+    <p>
+      {% blocktranslate trimmed %}
+        In that case, you can make a report directly to any or all of the current
+        (vice/co) chairs of the Code of Conduct committee. Their e-mail addresses are
+        listed on the <a href="{{ coc_teams_url }}">Code of Conduct Committee</a>
+        page. The chairs will follow the usual enforcement process with the other
+        members, but will exclude the member(s) that the report concerns from any
+        discussion or decision making.{% endblocktranslate %}</p>
+  {% endwith %}
```

Comment:
```suggestion
  <p>
    {% blocktranslate trimmed %}
      In that case, you can make a report directly to any or all of the current
      (vice/co) chairs of the Code of Conduct working group. Their e-mail addresses are
      listed on the <a href="{{ teams_url }}#code-of-conduct-team">Code of Conduct working group</a>
      page. The chairs will follow the usual enforcement process with the other
      members, but will exclude the member(s) that the report concerns from any
      discussion or decision making.{% endblocktranslate %}</p>
```

> Their e-mail addresses are listed on the...

This is actually not true of the teams page. Is this still true that this is a way to report? (cc @jefftriplett )